aws-java-sdk-dynamodb/pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>com.amazonaws</groupId>
     <artifactId>aws-java-sdk-pom</artifactId>
-    <version>1.11.762</version>
+    <version>1.11.762-SNAPSHOT</version>
```

Comment:
Set the version to `1.11.762-SNAPSHOT` so the maven release plugin will treat it appropriately.

Reply:
changed to `1.11.762-dw1-SNAPSHOT`